### PR TITLE
Team 5 Tutorial

### PR DESCRIPTION
**Pull request by drewlar**
Images and Markdown files for team 5

### CODE REVIEW

File: content/tutorials/Team5.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
+---
+title: Servos with ESP32 DevBoard
+date: 2024-12-06
+authors:
+  - name: Andrew Lara
```
Comment (AdinAck) [inline content/tutorials/Team5.md:5]: Nobody else? Your teammates (which are required to help) did not? Are you sure?

**Review by AdinAck — COMMENTED** [review]
Awesome :)

File: content/tutorials/Team5.md (hunk 1 above)
Comment (drewlar) [reply content/tutorials/Team5.md:5]: Changes have been committed!

Comment (AdinAck) [reply content/tutorials/Team5.md:5]: Okey dokey!

**Pull request merged**